.github/workflows/main.yml

```diff
@@ -92,10 +92,11 @@ jobs:
           LIBVCX_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx.dockerfile') }}
           CODECOV_DOCKERFILE_HASH=${{ hashFiles('ci/libvcx-ubuntu.dockerfile') }}
           LIBVCX_NODE_WRAPPERS_HASH=${{ hashFiles('wrappers/node') }}
+          LIBVCX_NODE_AGENT_HASH=${{ hashFiles('agents/node') }}
           JAVA_WRAPPERS_HASH=${{ hashFiles('wrappers/java') }}
           POOL_DOCKERFILE_HASH=${{ hashFiles('ci/indy-pool.dockerfile')}}
 
-          LIBVCX_HASH=${LIBVCX_SOURCE_HASH:0:15}-${LIBVCX_NODE_WRAPPERS_HASH:0:15}-${LIBVCX_DOCKERFILE_HASH:0:15}
+          LIBVCX_HASH=${LIBVCX_SOURCE_HASH:0:15}-${LIBVCX_NODE_WRAPPERS_HASH:0:15}-${LIBVCX_NODE_AGENT_HASH}-${LIBVCX_DOCKERFILE_HASH:0:15}
```

Comment:
Just a reminder: shorten the length because of the[ cache key length limit](https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#using-the-cache-action), and fix the alignment.